### PR TITLE
Adding sentry-url argument for neon-proxy.

### PR DESCRIPTION
**Pull request by KlimentSerafimov**
Added setry-url arg in /neon-proxy/templates/deployments.yaml for purposes of: https://github.com/neondatabase/neon/pull/2926
Updated version 1.2.6 -> 1.3.0. 
Auto-generated README.md via helm-docs.

### CODE REVIEW

File: charts/neon-storage-broker/README.md [hunk 1]
```diff
@@ -64,4 +64,4 @@ Kubernetes: `^1.18.x-x`
 | tolerations | list | `[]` | Tolerations for pod assignment. |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```
Comment (lassizci) [inline charts/neon-storage-broker/README.md:67]: This one causes the `ct` to detect the chart has changed

File: charts/neon-proxy/README.md [hunk 2]
```diff
@@ -60,8 +60,9 @@ Kubernetes: `^1.18.x-x`
 | settings.authBackend | string | `"link"` | auth method used (console|link|postgres) |
 | settings.authEndpoint | string | `""` | auth endpoint, e.g. "http://console.neon/authenticate_proxy_request/" |
 | settings.domain | string | `""` | domain used in TLS cert for client postgres connections |
+| settings.sentryURL | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
 | settings.uri | string | `""` |  |
 | tolerations | list | `[]` | Tolerations for pod assignment. |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```
Comment (lassizci) [inline charts/neon-proxy/README.md:68]:
```suggestion
Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
```

File: charts/neon-storage-broker/README.md (hunk 1 above)
Comment (KlimentSerafimov) [reply charts/neon-storage-broker/README.md:67]: I see! Thanks!

File: charts/neon-proxy/README.md (hunk 2 above)
Comment (lassizci) [reply charts/neon-proxy/README.md:68]: This is pretty annoying feature of helm-docs. We should probably upgrade to the latest version, but in the meantime this is probably the way to go.

Comment (KlimentSerafimov) [reply charts/neon-proxy/README.md:68]: yeah I was getting stuck in a catch-22 situation with helm-docs

> Documentation not up to date. Please run helm-docs and commit changes!

and after running helm-docs, it changed the version in the neon-storage-broker as well.

**Pull request merged**